Core/include/Acts/Seeding/InternalSpacePoint.hpp

```diff
@@ -42,6 +53,27 @@ class InternalSpacePoint {
   const float& varianceZ() const { return m_varianceZ; }
   const float& quality() const { return m_quality; }
   const float& cotTheta() const { return m_cotTheta; }
+  float topHalfStripLength() const {
+    return m_doubleMeasurement.topHalfStripLength;
+  };
+  float bottomHalfStripLength() const {
+    return m_doubleMeasurement.bottomHalfStripLength;
+  };
+  const Acts::Vector3 topStripDirection() const {
+    return m_doubleMeasurement.topStripDirection;
+  }
+  const Acts::Vector3 bottomStripDirection() const {
+    return m_doubleMeasurement.bottomStripDirection;
+  }
+  const Acts::Vector3 stripCenterDistance() const {
+    return m_doubleMeasurement.stripCenterDistance;
+  }
+  const Acts::Vector3 bottomStripCenterPosition() const {
+    return m_doubleMeasurement.bottomStripCenterPosition;
+  }
```

Comment:
I don't think returning `const` values is necessary.

Reply:
done